src/coordinates.h

```diff
@@ -44,26 +44,26 @@ struct real_coords {
         abs_pos = point( absx, absy );
 
         if( absx < 0 ) {
-            abs_sub.x = ( absx - 11 ) / 12;
-            sub_pos.x = 11 - ( ( normx - 1 ) % 12 );
+            abs_sub.x = ( absx - SEEX + 1 ) / SEEX;
```

Comment:
```suggestion
            abs_sub.x = ( absx - SEEX - 1 ) / SEEX;
```

Reply:
This preserves the previous value, or was the old value erroneous?

Reply:
This would change expression from  `( absx - 11 ) / 12` to `( absx - 13 ) / 12` which would be wrong.

Reply:
you are correct, stupid OOO and signs.